x-pack/heartbeat/monitors/browser/config.go

```diff
@@ -16,6 +17,7 @@ func DefaultConfig() *Config {
 	return &Config{
 		Sandbox:     false,
 		Screenshots: "on",
+		Timeout:     14 * time.Minute,
```

Comment:
@andrewvc should I leave the default option as 15 min already? Or do we want to change that once it's actually able to run to 15 min?

Reply:
14 min sounds weird to me, IMO we can keep it as whole number 15m and increase the service timeout. WDYT?